tests/unit/lang/format_print_will_end/CMakeLists.txt

```diff
@@ -1,12 +1,13 @@
-
 # define the test executable
-add_executable(format_print_will_end_tests tests.cpp ../../../../src/lang/format_print_will_end.cpp ../../../../src/lang/translator.cpp)
+add_executable(
+  format_print_will_end_tests ../../../../src/lang/format_print_will_end.cpp
+                              ../../../../src/lang/translator.cpp tests.cpp
+  )
 
 # define required search paths
 target_include_directories(format_print_will_end_tests PUBLIC . ${CMAKE_SOURCE_DIR}/src/lang)
 
-# define custom macros
-# target_compile_definitions(format_print_will_end_tests PUBLIC)
+# define custom macros target_compile_definitions(format_print_will_end_tests PUBLIC)
```

Comment:
Merging comments? Seriously? In this case it has its meaning not-to merge them.

Reply:
solved by https://github.com/prusa3d/Prusa-Firmware-Buddy/pull/2537